app/views/users/show.html.erb

```diff
@@ -0,0 +1,45 @@
+<div class="user_show container">
+
+  <div class="user_item row">
+    <div class="u_img">
+      <img src=<%= @user.photo %> alt="User Image">
+    </div>
+
+    <div class="user_cont">
+      <a href=<%= user_url(@user.id) %>>
+        <div class="user_desc col">
+          <h2 class="user_name"><%= @user.name %></h2>
+          <p class="user_post_n">Number of posts: <%= @user.posts_count %></p>  
+        </div>
```

Comment:
- [ ] Please use correct table column name for now you using `posts_count` but in database table column name is `post_counter` so kindly fix because its give an error :slightly_smiling_face: